app.js

```diff
@@ -1,30 +1,32 @@
-import express from "express";
-import morgan from "morgan";
-import cors from "cors";
-import path from "path";
-import "dotenv/config";
-import "./db/sequelize.js";
+import express from 'express';

```

Comment:
Could we add prettier?
Or maybe sync with team and create some task for it, it's quite hard to maintain formatting without it.
@bit-to-bit @Oleksandra-Mu Please, check